### PR TITLE
Add Travis badge

### PR DESCRIPTION
**Pull request by apupier**
Signed-off-by: Aurélien Pupier <apupier@redhat.com>

### CODE REVIEW

**Comment by apupier** [comment]
badge can be seen on https://github.com/apupier/teiid-language-server/tree/AddTravisBadge
Screenshot from 2020-03-17 14-56-34

**Comment by lhein** [comment]
image

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+![Build Status](https://travis-ci.org/teiid/teiid-language-server.svg?branch=master)](https://travis-ci.org/teiid/teiid-language-server)
```
Comment (lhein) [inline README.md]: something wrong with your brackets

**Review by rareddy — APPROVED** [review]
LGTM

**Review by lhein — CHANGES_REQUESTED** [review]
there seems to be an issue with the brackets

File: README.md (hunk 1 above)
Comment (apupier) [reply README.md]: well spotted. I badly resolved conflicts.

Comment (apupier) [reply README.md]: this is fine again when accessing  https://github.com/apupier/teiid-language-server/tree/AddTravisBadge

**Review by lhein — APPROVED** [review]

**Pull request merged**